src/test.coffee

```diff
@@ -79,7 +69,7 @@ class Test
 
     # Body
     if @response.schema
-      schema = @parseSchema @response.schema
+      schema = @response.schema
```

Comment:
Actually, I changed schema to be an array of objects (parsed schemas). See my note on add-tests.coffee. Should probably have changed the field's name to schemas (plural) to make that clearer.